tests/receivers/postgresql/testdata/resource_metrics/bundled.yaml

```
@@ -0,0 +1,121 @@
+resource_metrics:
+  - attributes:
+      container.image.name: <ANY>
+      container.name: <ANY>
+      postgresql.database.name: <ANY>
+    scope_metrics:
+      - instrumentation_scope:
+          attributes: {}
+          name: otelcol/postgresqlreceiver
+          version: <ANY>
```

```suggestion
          version: <VERSION_FROM_BUILD>
```

nit, though there's a handful of other instances as well